index.js

```diff
@@ -6,12 +6,17 @@ var app = express();
 var mongoose = require('mongoose');
 mongoose.connect('mongodb://localhost/airplanes');
 
+app.use(express.static(__dirname + '/public/'));
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(require('morgan')('dev'));
 
 app.use('/api/airplanes', require('./controllers/airplane'));
 
+app.get('/*', function(req, res) {
+  res.sendFile(__dirname + '/public/index.html');
+});
```

Comment:
This is catching every path and rendering the main index file for the home route. If you are trying to display an edit page it wont render.